src/ControllerMessenger.ts

```diff
@@ -325,8 +326,8 @@ export class RestrictedControllerMessenger<
    *
    * The event type being cleared *must* be in the current namespace.
    *
-   * @param eventType - The event type. This is a unique identifier for this event.
-   * @template E - A type union of Event type strings that are namespaced by N.
+   * @param event - The event type. This is a unique identifier for this event.
+   * @template E- A type union of Event type strings that are namespaced by N.
```

Comment:
Same, seems like the space should be kept?

Reply:
Thanks! This has been fixed.